tasks/preparation-provisioning.yaml

```
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: preparation-provisioning
+spec:
+  params:
+   - name : BONFIRE_IMAGE
+     type: string
+     description: The container Bonfire image to use for the tekton tasks
+   - name: CUSTOM_IMAGE
+     type: string
+     description: The container image to use for the provisioning setup task
+     default: quay.io/envision/provisioning-cicd:latest
+   - name: NS_REQUESTER
+     type: string
+     description: The name of the person/pipeline that requested the namespace
+   - name: EPHEMERAL_ENV_PROVIDER_SECRET
+     type: string
+     default: ephemeral-env-provider
+     description: "Secret for connecting to ephemeral env provider cluster"
+   - name: AWS_ACCOUNT_ID
+     type: string
+     description: The name of the Secret containing the AWS Account ID
+     default: aws-secret
```

I'd go with bit more verbose name for the secret, just because it's in a provisioning task so it's most likely provisioning specific. What about something like (I've left out the `-secret` suffix, but I don't feel strongly about that): 
```suggestion
     default: provisioning-aws-account-id
```